mailbox/api/src/main/java/org/apache/james/mailbox/model/MailboxAnnotationKey.java

```diff
@@ -31,16 +31,20 @@
 
```

Comment:
It belongs to the RFC at "3.2.  Namespace of Entries".
It said: "Entry names can have any number of components starting at 2, unless they fall under the vendor namespaces (i.e., have a /shared/vendor/<vendor-token> or /private/vendor/<vendor-token> prefix as described below), in which case they have at least 4 components."